src/pages/[platform]/build-a-backend/add-aws-services/analytics/existing-resources/index.mdx

```diff
@@ -0,0 +1,187 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Use existing AWS resources',
+  description: 'Configure the Amplify Libraries to use existing Amazon Pinpoint resources by referencing them in your configuration.',
+  platforms: [
+    'swift',
+    'android',
+    'flutter',
+    'javascript',
+    'react-native',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+  ],
+  canonicalObjects: [
+    {
+      platforms: [
+        'javascript',
+        'react-native'
+      ],
+      canonicalPath: '/[platform]/build-a-backend/add-aws-services/analytics/existing-resources/'
+    },
+    {
+      platforms: [
+        'react',
+        'angular',
+        'nextjs',
+        'vue'
+      ],
+      canonicalPath: '/[platform]/build-a-backend/add-aws-services/analytics/existing-resources/'
+    }
+  ]
```

Comment:
How did you figure out which platforms to group? Usually we group them based on the content being duplicates

If this was just done arbitrarily then I would suggest you just delete it for now

```suggestion
  ]
```